helm-charts/support/values.yaml

```diff
@@ -1,3 +1,8 @@
+prometheusAuth:
```

Comment:
The naming `prometheusAuth` is helpful, but I don't think it is helpful enough. We who use this configuration this often need to know more technical details and understand what this influence. I think a more detailed description is favorable, hiding less of the technical details and by that also giving more insights into what is done by configuring this.

I got confused about `prometheusAuth` as if it was how you configured prometheus to require authentication, but in reality the strategy is coupled with Prometheus creating an Ingress resource to be managed by nginx-ingress which in turn is hardcoded to reference a k8s Secret named `prometheus-ingress-auth` for username/password credentials required for requests.

I think this name should reflect that this is related to a) prometheus authentication, b) it is a k8s Secret being created, c) it is referenced by an Ingress resource, and d) the ingress resource is managed by nginx-ingress.

`prometheusAuthIngressSecret` perhaps?

Reply:
@consideRatio flawless logic and very well written feedback! I've renamed it to be prometheusIngressAuthSecret (since it's the prometheusIngress we are providing auth to, not prometheus itself).